design/cryptography.md

```diff
@@ -28,21 +28,22 @@ An additional complexity is that Obscuro has the requirement to support temporar
 - Generation happens when the central sequencer bootstraps the network
 - The Master Seed is shared across Validators using the process described in the [whitepaper](https://whitepaper.obscu.ro/obscuro-whitepaper/amalgamated.html#sharing-the-master-seed)
 
-## 2. Ensure Enclave Encrypted Communication
-- Client-Enclave communication is encrypted using the Obscuro Key and the Client Key
-- Clients encrypt using the Obscuro Public Key
-- Enclave responds encrypting with the Client Public Key ( also known as Viewing Keys )
-- The Obscuro Key is a Key-Pair deterministically calculated from the Master Seed + Genesis Rollup
-- The Public key will be published to the Management contract and used by all obscuro tooling ( like the wallet extension ) to encrypt transactions
-- Enclaves will determine the Private key when calculating the Master Seed + Genesis Rollup 
-  - Enclaves have the Master Seed through the attestation process
-  - Enclaves fetch Genesis Rollup through the Layer 1 blocks
+## 2. Client-Enclave Encrypted Communication
+- Client-Enclave communication is encrypted using both Obscuro Keys and the Client Keys
+- Obscuro Key and Client (Viewing Key) are both asymmetric key pairs
+  - The Private Obscuro Key is deterministically calculated from Master Seed + Genesis Rollup
```

Comment:
I don't think we need the genesis rollup.
There is good enough entropy in the MS.

Reply:
Removed 👍